resources/rpm/project.spec.erb

```diff
@@ -184,6 +184,26 @@ if [ "$1" -gt 1 ] ; then
   <%= get_preinstall_actions("upgrade") %>
 fi
 
+<%- install_triggers = get_install_trigger_scripts("install")
+ upgrade_triggers = get_install_trigger_scripts("upgrade")
+ -%>
```

Comment:
Previous comment about moving the logic out of the ERB if possible (it's usually possible)